README.md

```diff
@@ -9,17 +8,24 @@ An example output from OptView is shown below.
 
 ![Example](doc/OptView.png)
 
-Documentation
--------------
+## Optimizer Support
+pyOptSparse provides Python interfaces for a number of optimizers.
+However, we do not provide the source code for SNOPT and NLPQLP, due to their restrictive license requirements.
+Please contact the authors of the respective codes if you wish to obtain them.
+Furthermore, ParOpt and IPOPT are available as a open source package but must be installed separately.
```

Comment:
Link to the repos/sources for these optimizers

Reply:
Links are in the docs page for each optimizer.